packages/testlab/src/test-sandbox.ts

```diff
@@ -37,11 +38,16 @@ export class TestSandbox {
    * Will create a directory if it doesn't already exist. If it exists, you
    * still get an instance of the TestSandbox.
    *
-   * @param path - Path of the TestSandbox. If relative (it will be resolved relative to cwd()).
+   * @param rootPath - Path of the TestSandbox. If relative it will be resolved
+   * against the current directory.
+   * @param subdir - Subdirectory for the sandbox. If not provided, the sandbox
+   * will automatically generate a uuid as the name of the subdirectory.
```

Comment:
Please mention that it's possible to set `subdir` to an empty string to disable the new subdir behavior.

Maybe add the examples from the commit message?

```ts
// Create a sandbox as a subdirectory named with a uuid under the rootPath
const sandbox = new TestSandbox(rootPath);

// Create a sandbox in the root path directly
// This is same as the old behavior
const sandbox = new TestSandbox(rootPath, '');
const sandbox = new TestSandbox(rootPath, '.');

// Create a sandbox in the `test1` subdirectory of the root path
const sandbox = new TestSandbox(rootPath, 'test1');
```